src/components/UI/Notification.js

```diff
@@ -0,0 +1,38 @@
+import { useEffect } from "react";
+import classes from "./Notification.module.css";
+
+const Notification = ({
+  id,
+  title,
+  message,
+  status,
+  autoCloseDuration = 3000,
+  onClose,
+}) => {
+  useEffect(() => {
+    const timerId = setTimeout(() => {
+      onClose(id);
+    }, autoCloseDuration);
+    return () => {
+      clearTimeout(timerId);
+    };
+  }, [id, onClose, autoCloseDuration]);
```

Comment:
Notification is hidden automatically after some time. Maybe it would be better to implement this via a thunk?